docs/source/en/main_classes/quantization.md

```diff
@@ -128,12 +128,22 @@ For 4-bit model, you can use the exllama kernels in order to a faster inference
 
 ```py
 import torch
-gptq_config = GPTQConfig(bits=4, disable_exllama=False)
+gptq_config = GPTQConfig(bits=4)
+model = AutoModelForCausalLM.from_pretrained("{your_username}/opt-125m-gptq", device_map="auto", quantization_config = gptq_config)
+```
+
+With the release of the exllamav2 kernels, you can get faster inference speed compared to the exllama kernels. You just need to 
+pass `use_exllama_v2=True` in [`GPTQConfig`] and disable exllama kernels:
+
+```py
+import torch
+gptq_config = GPTQConfig(bits=4, use_exllama_v2=True, disable_exllama=True)
```

Comment:
pretty obvious that if you `use_exllama_v2` exllama should be disabled by default wdyt? (a comment could say that it's disabled by default if you set `use_exllasma_v2`

Reply:
Works for me !